Common/Util/SeriesJsonConverter.cs

```diff
@@ -49,6 +49,9 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
                 values = series.Values;
             }
 
+            // have to add the converter we want to use, else will use default
+            serializer.Converters.Add(new ColorJsonConverter());
```

Comment:
Interesting that it doesn't use the right converter since it's decorated on the memory.